apps/prairielearn/src/api/trpc/routers/course-files/index.ts

```diff
@@ -1,11 +1,11 @@
 import { router } from '../../trpc.js';
 
+import { batchDeleteQuestions } from './batchDeleteQuestions.js';
 import { createQuestion } from './createQuestion.js';
-import { deleteQuestion } from './deleteQuestion.js';
 import { updateQuestionFiles } from './updateQuestionFiles.js';
 
 export const courseFilesRouter = router({
   createQuestion,
   updateQuestionFiles,
-  deleteQuestion,
+  batchDeleteQuestions,
```

Comment:
Nit, I'd name this `deleteQuestions` ("batch" is sort of implicit in the plural "questions").

Reply:
Done in 253c7cd.